requirements.txt

```diff
@@ -1,3 +1,5 @@
 ## Requirements
```

Comment:
I prefer to set versions in the requirements files. Maybe you could add the version to all of them ;)

Reply:
Done, I will take that into account from now on :+1: